packages/mutation-testing-elements/package.json

```diff
@@ -26,5 +26,9 @@
   "devDependencies": {
     "mutation-testing-metrics": "^1.2.2",
     "mutation-testing-report-schema": "^1.2.0"
+  },
+  "dependencies": {
+    "@types/prismjs": "^1.16.0",
+    "prismjs": "^1.19.0"
```

Comment:
Is this really a dependency? We're not bundling it with webpack?

Reply:
EDIT: Since we're importing from `prism`, I guess this is bundled. Please make `prismjs` itself a dev dependency

Reply:
I'll change both to devDependencies